frontend/src/pages/MessagingQueues/MessagingQueues.tsx

```diff
@@ -192,7 +192,7 @@ function MessagingQueues(): JSX.Element {
 							</Button>
 						</div>
 					</div>
-					<div className="overview-info-card">
+					<div className="overview-info-card middle-card">
 						<div>
```

Comment:
Empty div ?

Reply:
using these to categorise the child elements - `[(1,2) ---- 3]`